.github/workflows/terraform-lint.yml

```
@@ -0,0 +1,25 @@
+name: terraform-lint
+
+on:
+    push:
+        branches:
+            - main
+        paths:
+            - '**.tf'
+    pull_request:
+        branches:
+            - main
+        paths:
+            - '**.tf'
+  
+
+jobs:
+    delivery:
+
+      runs-on: ubuntu-latest
+
+      steps:
+      - name: Check out code
+        uses: actions/checkout@main
+      - name: Lint Terraform
+        uses: actionshub/terraform-lint@main
```

Always pin specific versions.
```suggestion
      - name: Check out code
        uses: actions/checkout@v4
      - name: Lint Terraform
        uses: actionshub/terraform-lint@v2
```
Not sure about the right version to use for `terraform-lint`, but their latest release is `2.0.1`, so `v2` may work.